packages/transmute-adapter-firestore/__tests__/interface.spec.js

```diff
@@ -1,13 +1,14 @@
 const TransmuteAdapterFirestore = require("../index");
 
-const admin = require("firebase-admin");
+// const admin = require("firebase-admin");
```

Comment:
do we want to leave commented out code?

Reply:
In this case I feel we do. 

The issue is that the mocked tests do not help an implementer without these comments...

Reply:
regarding gitignore, it should be present everywhere imo.